apps/adoption/adoption-cron-draft-case-deletion-alert/adoption-cron-draft-case-deletion-alert.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: adoption-cron-draft-case-deletion-alert
+spec:
+  releaseName: adoption-cron-draft-case-deletion-alert
+  values:
+    job:
+      environment:
+        TASK_NAME: AlertDraftCaseApplicantBeforeDeletion
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/adoption/cos-api:pr-677-831ffd1-20230209115325 #{"$imagepolicy": "flux-system:adoption-cos-api"}
```

Comment:
Is this correct image policy ?

Reply:
Yeah. We are using different PR related image. Instead of demo branch image. The similar change is the below file

https://github.com/hmcts/cnp-flux-config/blob/master/apps/adoption/adoption-cos-api/demo.yaml

Reply:
Review `imagepolicy`, flux will merge with production image and now pr-*

Reply:
@thomast1906  Corrected image policy